nengo_loihi/dvs.py

```diff
@@ -14,7 +14,7 @@ class DVSFileChipProcess(ChipProcess):
     ----------
     file_path : string
         The path of the file to read from. Can be a ``.aedat`` or ``.events`` file.
-    kind : "aedat" or "events" or None, optional
+    format : "aedat" or "events" or None, optional
```

Comment:
It would be good to use one set of terminology everywhere; is there an external spec or something that we are trying to match? If not, I would want to avoid the use of `format` since it is a Python builtin. I'm not a huge fan of `kind` either, could we do `file_fmt`? `fmt` is typically use for a string formatting template.

Reply:
`format` is a built-in, but as I said above, we never use it. There are also examples in Nengo/Numpy/TensorFlow where built-ins are overridden locally.

But I'm also fine with `file_fmt`, so I'll change it to that.